src/iop/bilateral.cc

```diff
@@ -69,9 +69,9 @@ const char *name()
   return _("denoise (bilateral filter)");
 }
 
-int groups()
+int groups() 
 {
-  return IOP_GROUP_CORRECT;
+return dt_iop_get_group("input color profile", IOP_GROUP_CORRECT);
```

Comment:
the name was wrong, cut&paste... I have fixed that.

Reply:
And there was a missing include, seems like you didn't even test to compile the code!

Reply:
Actually not, sorry. I'm not  experienced in DT and I didn't want to make something wrong

Reply:
You should have said so upfront I would have paid more attention to the code. Anyway all under control and working now.

Reply:
You're definitely right. I will. I promise